driver-async/src/main/com/mongodb/async/client/AggregateIterableImpl.java

```diff
@@ -71,7 +71,8 @@
                           final ReadConcern readConcern, final WriteConcern writeConcern, final OperationExecutor executor,
                           final List<? extends Bson> pipeline, final AggregationLevel aggregationLevel) {
         super(clientSession, executor, readConcern, readPreference);
-        this.operations = new AsyncOperations<TDocument>(namespace, documentClass, readPreference, codecRegistry, writeConcern, false);
+        this.operations = new AsyncOperations<TDocument>(namespace, documentClass, readPreference, codecRegistry, writeConcern,
+                true, true);
```

Comment:
Why change retryWrites to true?

Reply:
It might actually be correct, in that with a $out stage an aggregation is actually a write operation.  But if so, the value needs to be plumbed in through the constructor.  Let's handle that in a separate commit.

Reply:
Ok. I have reverted back to `false`.